### PR TITLE
Service

### PR DESCRIPTION
**Pull request by InnaKushnir**

### CODE REVIEW

**Comment by InnaKushnir** [comment]
Last commit with BookService solution.

File: alembic/README [hunk 1]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (EdAlekseiev) [inline alembic/README:1]: Add empty line, please

File: main.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    return {"message": "Hello World"}
+
+
+@app.get("/authors/", response_model=List[schemas.Author])
+def get_author(book_service: crud.BookService = Depends()):
+    return book_service.get_all_author()
+
+
+@app.post("/authors/", response_model=schemas.Author)
+def create_author(author: schemas.AuthorCreate, book_service: crud.BookService = Depends()):
```
Comment (EdAlekseiev) [inline main.py]: This method must take paginations value as query parameters. Please, add it.

File: main.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    if db_author is None:
+        raise HTTPException(status_code=404, detail="Author not found")
+
+    return db_author
+
+
+@app.get("/books/", response_model=List[schemas.Book])
+def get_books(author_id: Union[int, None] = None, book_service: crud.BookService = Depends()):
+    return book_service.get_books(author_id=author_id)
```
Comment (EdAlekseiev) [inline main.py]: This method must take paginations value as query parameters. Please, add it.

File: main.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+
+@app.post("/books/", response_model=schemas.Book)
+def create_author(book: schemas.BookCreate, book_service: crud.BookService = Depends()):
+
```
Comment (EdAlekseiev) [inline main.py:57]: Delete this empty line.

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

File: fast.db
Comment (Bohdan-Salamakha) [inline fast.db:1]: remove this file from github

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
I got an error
image
image

fix it

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
not fixed
you did PR from the service branch
so work on it, not on pythons

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
fix errors when I go to the /books/ endpoint
image
image

**Pull request closed without merging**